events/fair-ml.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: "Fair ML Speaker Series"
+permalink: "/fair-ml"
+---
+
+<h1 class="title page-title"> Fair ML Speaker Series</h1>
+
+<p style = "font-size: 115%;">
+	Thanks to our <a class="tla-link" href="{{site.baseurl}}/speaker-series"> Speaker Series Team </a>, our Fair ML Speaker Series was a success! <b> Learn all about it on this page. </b>
+
+</p>
+
+<hr class="divider" />
+
+<div class="citylab-section" style = "margin-top: 2em;">
+	<div class = "grid-col">
+		<img class = "img-responsive" src = "{{site.baseurl}}/img/undraw/content.svg" alt = "drawing related to content" />
+	</div>
+	<div>
+		<h2 class = "text-2x"> Why does fair machine learning matter?
+		</h2>
+		<p style = "font-size: 115%;">
+			Have you ever wondered... Can AI discriminate? Of course! Thanks to Catherine Yeo, we were inspired to teach others about fair machine learning. Check out her work below to learn more.
+		</p>
+
+		<ul style = "font-size: 150%;">
```

Comment:
We've already defined a class that does this (or something similar) - `.text-15x`!